### ISSUE TITLE
Remove custom styling for youtube videos?

### ISSUE THREAD

**Issue by chovy** [sampotts/plyr#1754]
YouTube has informed me that the default video player cannot be customized. I still need plyr to disable simultaneous video playing which works, but can I remove the custom plyr decorations somehow?

**Comment by sampotts** [comment]
I guess it may be time to remove support for custom controls with YouTube and Vimeo (unless Pro/Plus) given this. A few other people have had the same feedback. Even their own advice allows you to customize controls but they must not overlay the player.

**Comment by chovy** [comment]
Yeah i think the main issue they had was the blue play button over the video. They flat out told me not to customize the player though, I can try to re-submit if we can hide the play button overlay.

**Comment by chovy** [comment]
@sparkktv yes I'm using youtube api, and they wanted to see the player setup, they said I need to remove the plyr overlay, you cannot block the youtube video according to their requirements.

**Comment by sparkktv** [comment]
@chovy Correct while using their API, but again they have no knowledge that you are not using their player if you are not accessing their API at the same time. 
So my site, I use Plyr on the website itself but our Amazon app requires their API to be used and on their we use their Youtube Player by default. Because Plyr is using the normal embed that Youtube uses and just using a custom skin on top, Youtube will still think it's their own player. And using a custom player skin is not against their TOS only their API TOS. Video.js being the most popular of all used on many sites.
Using their API though, they control the usage and can say what you can't or can do and some usage requires that. And yes they only require that material/screenshot of how you are using Youtube when you need a quota increase. Most of us will never be affected by that.
And I might be wrong but I think there is already a way to turn plyr off and show the Youtube player just by using the enabled flag/command and turning it to false.

**Comment by sparkktv** [comment]
@sampotts It seems after I did some digging & research that Youtube only asks for supporting materials and how you are using YouTube when you need an API quota increase. This will likely never affect most users using plyr with YouTube. 
Possibly you should put a disclaimer that Plyr should not be used along with Youtube API. Most people never need to use the API, only if you are developing something that requires it.

**Comment by chovy** [comment]
@sparkktv can you give me an example of disabling it?

**Comment by sparkktv** [comment]
@chovy If you use the iframe method you can use the enabled:false tag to show the Youtube Player instead of the Plyr skin. Using the div non-progressive mode though it can not be done. My only suggestion would be to add a method to turn the Plyr skin on and off for both methods.
<img width="2048" alt="Screen Shot 2020-04-04 at 2 40 53 PM" src="https://user-images.githubusercontent.com/23144931/78458820-4e515300-7682-11ea-8365-15c84163e2fe.png">

**Comment by chovy** [comment]
That works but now I can't stop the sibling videos when play is clicked.

**Comment by jeroenbell** [comment]
+1

Received an audit by YouTube and they are demanding to remove the custom styling. The problem I'm having is that I can't properly use the iframe embed because of usability issues (we're allowing users to drag the video wherever they want, so we need an overlay to capture mouse events). Only solution is to copy their controls 1:1 and use the API to control the video.
